### DISCUSSION TITLE
How I get the member who moved another member?

### DISCUSSION THREAD

**Discussion by michel929** [discord-jda/JDA#2345] [Questions and Help] [answered]
Before update 5 there was the PlayerMoveEvent but now there is only the GuildVoiceUpdateEvent how do you get the member who moved another in this event? Or is there another event for this?

Thx :)

**Comment by MinnDevelopment** [comment, marked as answer]
There never was a way to get this information from the event. The previous event did not have this either. You have to use the guild audit logs to find out which moderator moved members.

**Reply by michel929** [reply]
Ok thx :)
